beginend.el

```diff
@@ -232,6 +232,32 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+;; Copy/pasted from expand-region
+(defun beginend--point-is-in-comment-p ()
+  "Return non-nil if point is in comment."
+  (or (nth 4 (syntax-ppss))
+      (memq (get-text-property (point) 'face)
+            '(font-lock-comment-face font-lock-comment-delimiter-face))))
```

Comment:
I have no clue about this area of Emacs source code so I'm ok to blindly trust you. Nevertheless, the current code is extracted from expand-region and I never had any problem with it. Moreover, your function depends on `dash` which is not a dependency of beginend for now (replacing `-when-let` by `if-let` from `subr-x` would make Emacs 25 the minimum Emacs to use beginend). What do you recommend?